benchmarks/src/jvmMain/kotlin/pro/respawn/flowmvi/benchmarks/setup/fluxo/FluxoStore.kt

```diff
@@ -0,0 +1,19 @@
+package pro.respawn.flowmvi.benchmarks.setup.fluxo
+
+import kotlinx.coroutines.Dispatchers
+import kt.fluxo.core.annotation.ExperimentalFluxoApi
+import kt.fluxo.core.store
+import kt.fluxo.core.updateState
+import pro.respawn.flowmvi.benchmarks.setup.BenchmarkIntent
+import pro.respawn.flowmvi.benchmarks.setup.BenchmarkState
+
+@OptIn(ExperimentalFluxoApi::class)
+internal inline fun fluxoStore(
+) = store(BenchmarkState(), handler = { it: BenchmarkIntent ->
```

Comment:
Handler constructor is the slowest one in Fluxo :)
You need a direct reducer if you want performance. Use `reducer` argument name for this and something like `it.counter + 1` in lambda body.

<img width="606" alt="image" src="https://github.com/user-attachments/assets/f9bbae9b-31ed-427f-b213-3b473c32e6e6">